src/integrations/settings-integration.php

```diff
@@ -379,6 +379,23 @@ protected function get_script_data() {
 		$post_types             = $this->post_type_helper->get_indexable_post_type_objects();
 		$taxonomies             = $this->taxonomy_helper->get_indexable_taxonomy_objects();
 
+		// Check if attachments are included in indexation.
+		if ( ! \array_key_exists( 'attachment', $post_types ) ) {
+			// Always include attachments in the settings, to let the user enable them again.
+			$attachment_object = \get_post_type_object( 'attachment' );
+			if ( ! empty( $attachment_object ) ) {
+				$post_types['attachment'] = $attachment_object;
+			}
+		}
+		// Check if post formats are included in indexation.
+		if ( ! \array_key_exists( 'post_format', $taxonomies ) ) {
+			// Always include attachments in the settings, to let the user enable them again.
```

Comment:
> Always include attachments in the settings, to let the user enable them again.

Should be, Always include **post_format** in the settings, to let the user enable them again.

I'm applying that change myself